recipes/uavro/meta.yaml

```diff
@@ -0,0 +1,42 @@
+{% set name = "uavro" %}
+{% set version = "0.0.2" %}
+{% set sha256 = "0d6b6b55527cf23f41ff20314c9841829781d817847e869446729a85adf263af" %}
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  fn: {{ name }}-{{ version }}.tar.gz
+  url: https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/{{ name }}-{{ version }}.tar.gz
+  sha256: {{ sha256 }}
+
+build:
+  number: 0
+  script: python setup.py install --single-version-externally-managed --record record.txt
+
+requirements:
+  build:
+    - python
+    - setuptools
+    - cython
+    - numpy
+    - msinttypes  # [win]
+  run:
+    - python
+    - pandas
+    - fastparquet
+
+test:
+  imports:
+    - uavro
+
+about:
+  home: https://github.com/martindurant/uavro
+  license: MIT
+  license_family: MIT
```

Comment:
please include the `license_file` manually (https://conda-forge.org/docs/meta.html#packaging-the-license-manually). See also:

https://github.com/martindurant/uavro/issues/6